pcl_config.h.in

```diff
@@ -20,7 +20,15 @@
 #define PCL_VERSION_COMPARE(OP, MAJ, MIN, PATCH) \
     (PCL_VERSION*10+PCL_DEV_VERSION OP PCL_VERSION_CALC(MAJ, MIN, PATCH)*10)
 
-#cmakedefine HAVE_TBB 1
```

Comment:
What happened here?

Reply:
Spontaneous code deletion? Really sorry, I've got no clue how i did that.